SeaPublicWebsite/Views/Feedback/FeedbackForm.cshtml

```diff
@@ -30,21 +30,15 @@
                     m => m.WhatUserWasDoing, 
                     labelOptions: new LabelViewModel
                     {
-                        Text = "What were you doing?",
-                        Classes = "govuk-!-font-weight-bold"
```

Comment:
Just to be clear, if you need these styles then there should be equivalent ones without `!` that should also work (e.g. `govuk-font-weight-bold`). If those don't work then we may _have_ to use these.